icons/glitter.svg

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This icon should be modified according to our art guidelines. We'll take care of it.